Android/wear/src/main/java/dev/saxionroosters/models/Week.java

```diff
@@ -0,0 +1,45 @@
+package dev.saxionroosters.models;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * Created by Doppie on 24-2-2016.
+ */
+public class Week implements Serializable {
```

Comment:
Hmm het is jammer dat het nodig is om het model te kopiëren. Ik stel voor dat we het model in een aparte model 'backend' gooien inclusief de requests. Dan kan je die als dependency toevoegen aan je wear app en hetzelfde voor de mobile app. Maar dat is een refactor stap - iets waar we sowieso nog graag mee bezig willen gaan.